README.adoc

```diff
@@ -100,7 +100,7 @@ sudo dnf install @buildsys-build rust cargo cmake qt5-qtdeclarative-devel ibus-d
 ```
 On a Arch Linux / Arch Based system you can easily install them like this
 ```bash
-sudo pacman -Sy base-devel rust cmake qt5-base libibus
+sudo pacman -S base-devel rust cmake qt5-base libibus
```

Comment:
@chandradeepdey `-Syu` should be better.

Reply:
But it would be inconsistent with the Fedora and Ubuntu commands, where no check for updates is done. Further, -Syu can involve complications with breaking upgrades.
To be fair, the person executing the command should know what they are doing; but I think we should go with -S anyway.

Reply:
I am not an Arch user, so I don't know the details. But the article you provided has the following statement:
```
Always upgrade (with pacman -Syu) before installing a package. 
```

Reply:
I am not an Arch user, so I don't know the details. But the article you provided has the following statement:
```
Always upgrade (with pacman -Syu) before installing a package. 
```

Reply:
I am not an Arch user, so I don't know the details. But the article you provided has the following statement:
```
Always upgrade (with pacman -Syu) before installing a package. 
```

Reply:
@mominul the context of that statement is the act of just checking for updates. If you check for updates, you should perform a full system upgrade.
In the previous link there is also this text in the same section
> The bash script checkupdates, included with the pacman-contrib package, provides a safe way to check for upgrades to installed packages without running a system update at the same time.

https://wiki.archlinux.org/index.php/pacman#Installing_packages
The warning in red and the commands that follow should clear this up.

Reply:
> But it would be inconsistent with the Fedora and Ubuntu commands, where no check for updates is done.
> To be fair, the person executing the command should know what they are doing; but I think we should go with -S anyway.

Ok, fair points.
Arch repos get updated so often that the package url on disk cache may not exist any more in the repo depending on how long ago the use system updated (and the preferred mirror). But yes, the user should know what they're doing.
Let's go with the non-clever standard `-S` flag then.